src/components/ProcessCreate/Census/Web3.tsx

```diff
@@ -227,54 +226,20 @@ export const CensusWeb3Addresses = () => {
             name='weightedVote'
             defaultValue={weighted}
             render={({ field: { onChange, onBlur, value, ref } }) => (
-              <Checkbox
+              <DetailedCheckbox
                 onChange={(event: ChangeEvent<HTMLInputElement>) => setValue('weightedVote', event.target.checked)}
                 onBlur={onBlur}
-                ref={ref}
+                name={'weightedVote'}
                 isChecked={value}
-                variant={'radiobox'}
-              >
-                <Flex>
-                  <Icon as={BiCheckDouble} />
-                  <Text>
-                    <Trans i18nKey='form.process_create.weighted'>Weighted vote</Trans>
-                  </Text>
-                </Flex>
-                <Text>{t('form.process_create.spreadsheet.requirements.list_three')}</Text>
-              </Checkbox>
+                variant={'detailed'}
+                icon={<BiCheckDouble />}
+                title={'form.process_create.weighted'}
+                description={'form.process_create.spreadsheet.requirements.list_three'}
+              />
```

Comment:
Same here, we're duplicating code having the DetailedCheckbox inside a Controller.